front/lib/api/config.ts

```diff
@@ -116,6 +116,9 @@ const config = {
       "DEVELOPMENT_DUST_APPS_VAULT_ID"
     );
   },
+  getLookUpBearerToken: (): string | undefined => {
+    return EnvironmentConfig.getOptionalEnvVariable("LOOKUP_API_BEARER");
```

Comment:
Nit: for me "bearer" is the name of the http header, I would use LOOKUP_API_SHARED_SECRET or something like that instead.

Reply:
Also, LOOKUP_API is a bit vague.
REGION_RESOLVER or something like that ?